analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/SonarAnalyzer.RuleDescriptorGenerator.csproj

```diff
@@ -16,4 +16,13 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.3.2" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="1.3.2" />
   </ItemGroup>
+
+  <Target Name="CopyBinaries" AfterTargets="Build">
```

Comment:
Same method as we use to package analyzers themselves:

https://github.com/SonarSource/sonar-dotnet/blob/81d0d19f65d367a66d15f6c63b6d0e2fe6ea4490/analyzers/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj#L62-L70